### PR TITLE
#571 - Support for include-patterns in AddUpdatePermissionSlice

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #571 
Support for `"include-patterns"` in `AddUpdatePermissionSlice`

### CODE REVIEW

File: src/test/java/com/artipie/api/artifactory/AddUpdatePermissionSliceTest.java [hunk 1]
```diff
@@ -105,16 +107,33 @@ void updatesPermissions() throws IOException {
             this.permissionsForUser(repo, "john"),
             Matchers.containsInAnyOrder("*")
         );
+        MatcherAssert.assertThat(
+            "Sets patterns",
+            this.patterns(repo),
+            Matchers.contains("**")
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/api/artifactory/AddUpdatePermissionSliceTest.java:113]: @olegmoz I'd suggest to add some more patterns to test json to be sure all of them are stored correctly

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@olegmoz just one comment, take a look please

**Comment by olegmoz** [comment]
@olenagerasimova thanks, improved test as you suggested

**Comment by olenagerasimova** [comment]
@olegmoz thanks, good to merge

**Review by olenagerasimova — APPROVED** [review]

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#600` is not assigned, can't get performer

**Pull request merged**